### PR TITLE
Make scopes optional to match powershell

### PR DESCRIPTION
**Pull request by calebkiage**

### CODE REVIEW

**Review by zengin — APPROVED** [review]

**Review by peombwa — APPROVED** [review]

File: src/Microsoft.Graph.Cli.Core/Microsoft.Graph.Cli.Core.csproj [hunk 1]
```diff
@@ -14,15 +14,15 @@
     <Deterministic>true</Deterministic>
     <!-- Recommended: Embed symbols containing Source Link in the main file (exe/dll) -->
     <DebugType>embedded</DebugType>
-    <Version>0.1.1-preview.3</Version>
```
Comment (zengin) [inline src/Microsoft.Graph.Cli.Core/Microsoft.Graph.Cli.Core.csproj:17]: If the package upgrades are not relevant to the fix, it might be a good idea to have a separate PR for that. Keeping a clean history (one fix per PR) makes cherry-picking and reverting changes much easier.

Comment (calebkiage) [reply src/Microsoft.Graph.Cli.Core/Microsoft.Graph.Cli.Core.csproj:17]: Thanks @zengin, I will consider it in the next PRs.
